src/tests/test_module.py

```diff
@@ -613,9 +613,9 @@ def test_downsampling_rules(self):
     def test_automatic_timestamp(self):
         with self.redis() as r:
             assert r.execute_command('TS.CREATE', 'tester')
-        curr_time = int(time.time())
+        curr_time = int(time.time()*1000)
         r.execute_command('TS.ADD', 'tester', '*', 1)
-        result = r.execute_command('TS.RANGE', 'tester', 0, int(time.time()))
+        result = r.execute_command('TS.RANGE', 'tester', 0, int(time.time() * 1000))
         # test time difference is not more than 5 second
         assert result[0][0] - curr_time <= 5
```

Comment:
@gkorland 
```suggestion
        assert result[0][0] - curr_time <= 5000
```

Reply:
@danni-m I would have leave it at 5ms just change the comment, 5ms should be enough diff

Reply:
fixed comment to millisecond